src/components/bluebutton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, children, label, ...props }) =>
```

Comment:
`targetTo` is not a descriptive name for the button's on click handler. Why not `onClick` ?